src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/FailureReader.java

```diff
@@ -74,15 +74,15 @@ public FailureReader(Indication indication) {
      * is found in the log of the given build; return null otherwise.
      * @throws IOException if so.
      */
-    public abstract FoundIndication scan(AbstractBuild build) throws IOException;
+    public abstract FoundIndication scan(Run build) throws IOException;
```

Comment:
Same here, although we could take the risk tha no one else has implemented the extension point, but we all know what happens to assumptions ;)